README.md

```diff
@@ -21,18 +35,59 @@ Just make sure your changes and pull requests are in accordance with the [contri
+
 ## Building
 
-1. Clone this repo.
-2. Run `RUN_THIS.py` to init submodules and download the engine.
-3. Compile the solution.
+**Clone** this repo:
+```shell
+# Cloning using HTTPS repository access
+git clone https://github.com/{user}/space-station-14.git
+
+# Cloning using SSH repository access
+git clone git@github.com:{user}/space-station-14.git
+```
+
+> [!NOTE]
+> Replace `{user}` to your GitHub username. The easiest way to find it is in the GitHub service at the very top of the repository page.
+
+**Go** to the project folder and run `RUN_THIS.py` to initialize the sub-modules and load the engine:
+```shell
+cd space-station-14
+python RUN_THIS.py
+```
+Compile the solution: 
+
+Build the server and use `dotnet build` or `dotnet build --configuration Release` (if necessary).
 
 [More detailed instructions on building the project.](https://docs.spacestation14.com/en/general-development/setup.html)
 
 ## License
 
-All code for the content repository is licensed under [MIT](https://github.com/space-wizards/space-station-14/blob/master/LICENSE.TXT).
+<details>
+<summary><a href="#"><img src="https://img.shields.io/badge/licence-MIT-green?style=for-the-badge" alt="MIT license"></a></summary>
+
+>All code for the content repository is licensed under [MIT](https://github.com/space-wizards/space-station-14/blob/master/LICENSE.TXT).
+</details>
+
+<details>
+<summary><a href="#"><img src="https://img.shields.io/badge/licence-CC_3.0_BY--SA-lightblue?style=for-the-badge" alt="Creative Commons 3.0 BY-SA"></a></summary>
+
+>Most assets are licensed under [CC-BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) unless stated otherwise. Assets have their license and the copyright in the metadata file. [Example](https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/Objects/Tools/crowbar.rsi/meta.json).
+</details>
 
-Most assets are licensed under [CC-BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) unless stated otherwise. Assets have their license and the copyright in the metadata file. [Example](https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/Objects/Tools/crowbar.rsi/meta.json).
+<details>
+<summary><a href="#"><img src="https://img.shields.io/badge/licence-CC_3.0_BY--NC--SA-lightblue?style=for-the-badge" alt="Creative Commons 3.0 BY-SA"></a></summary>
 
-Note that some assets are licensed under the non-commercial [CC-BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) or similar non-commercial licenses and will need to be removed if you wish to use this project commercially.
+>Note that some assets are licensed under the non-commercial [CC-BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) or similar non-commercial licenses and will need to be removed if you wish to use this project commercially.
+</details>
```

Comment:
This part just kinda looks weird now, the shields look out of place. And It's not very obvious there's information behind them when clicking the arrow. I suggest reverting this,